Makefile

```diff
@@ -133,6 +134,16 @@ reproducibility-test-local-%:  ## Builds the specified target defined in the Pkg
 $(TARGETS):
 	@$(MAKE) docker-$@ TARGET_ARGS="--tag=$(REGISTRY_AND_USERNAME)/$@:$(TAG) --push=$(PUSH)"
 
+.PHONY: image
```

Comment:
let's drop these, since these are autogenerated, so running a `make rekres` would drop these

Reply:
I'll fix it up

Reply:
I can submit a PR to [the overlays page](https://www.talos.dev/v1.7/advanced/overlays/) documenting the docker command, because I was heavily relying on [this repository](https://github.com/milas/talos-sbc-rk3588).

Reply:
yes, that would be nice